aws/resource_aws_elasticache_cluster_test.go

```diff
@@ -1093,113 +1105,113 @@ resource "aws_elasticache_cluster" "bar" {
 }
 
 var testAccAWSElasticacheClusterInVPCConfig = fmt.Sprintf(`
-data "aws_availability_zones" "available" {
-  # InsufficientCacheClusterCapacity: cache.m1.small (VPC) is not currently supported in the availability zone us-east-1b
-  blacklisted_zone_ids = ["use1-az1"]
-  state                = "available"
+data "aws_availability_zones" "test" {
```

Comment:
Another one for "available" -- sorry to put these in a weird order, I'm reading bottom to top so I can look through the configs first.

Reply:
👍